src/infection-model/index.js

```diff
@@ -0,0 +1,122 @@
+// this function comes from Justine, adapted from python model
+function simulateOneDay({
+  x,
+  N,
+  R0,
+  dIncubation = 2,
+  dInfectious = 4.1,
+  dRecoveryMild = 9.9,
+  dHospitalLag = 2.9,
+  dHospitalRecovery = 22,
+  dHospitalFatality = 8.3,
+  pSevereCase = 0.26,
+  pFatalityRate = 0.026,
```

Comment:
For more context: These hard-coded values are what we are currently using to power the spreadsheet model currently except the "Normal Healthcare Fatality Rate", which is currently set to 1% on the variables tab but is actually closer to 2.6% [according to the CDC](https://www.cdc.gov/mmwr/volumes/69/wr/mm6912e2.htm). I've changed it in the working copy but not the live version yet.